### ISSUE TITLE
`multi_pose_sampling` does not write `summary.csv` and does return per residue pLDDT scores

### ISSUE THREAD

**Issue by jsture** [zrqiao/NeuralPLexer#12]
`summary.csv` is only generated by benchmarking functions and not the main predictor function. In addition, per residue scores are not returned. 

One would probably have to add:

```
if confidence:
    plddt, plddt_lig = model.run_confidence_estimation(
        sample, output_struct, return_avg_stats=True
    )

    sample = model.run_confidence_estimation(
        sample, output_struct, return_avg_stats=False
    )
```

Then append that to some more lists and write those two as .csv files. 

The logic and output of `run_confidence_estimation` is a bit too non-straightforward and undocumented for me to add this myself.

**Comment by Samuel-gwb** [comment]
Yes, it works fine just printing plddt and plddt_lig, while "sample" gives huge matrix setting return_avg_stats=False.

**Comment by amorehead** [comment]
I believe [this PR](https://github.com/zrqiao/NeuralPLexer/pull/27) addresses the missing plDDT values issue.
